packages/create-react-native-library/src/index.ts

```diff
@@ -657,6 +657,7 @@ async function create(_argv: yargs.Arguments<any>) {
     repo: repoUrl,
     example,
     year: new Date().getFullYear(),
+    local,
```

Comment:
```suggestion
```
We don't need to add a new property here, we can use `example === 'local'` instead.